.github/workflows/publish-helm-chart.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish Helm Chart
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    # only run this on push events, not in PRs
+    if: github.event_name == 'push' && github.repository == 'dagger/dagger'
```

Comment:
This GitHub Actions workflow only has a `workflow_dispatch` event, which I don't think will pass this `github.event_name == 'push'` condition. Do we want to publish this on e.g. `helm/chart/v` tag instead?

Reply:
I was keeping it from running automatically, so I could more easily test it. I'll change it before merging. I like your suggestion for the tag.